clients/python/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.7
+COPY . .
```

Comment:
I think we need to copy a `requirements.txt` in this directory to help with missing python dependencies in the container like `requests`. We would just need to also copy the `requirements.txt` into the container and run an install `pip3 install -r requirements.txt`.